pipeline/beam_tables.py

```diff
@@ -113,16 +101,24 @@ def _get_bigquery_schema(scan_type: str) -> Dict[str, Any]:
       'satellite' or 'blockpage'
 
   Returns:
-    A nested Dict with bigquery fields like SCAN_BIGQUERY_SCHEMA.
+    A nested Dict with bigquery fields like BASE_BIGQUERY_SCHEMA.
   """
   if scan_type == satellite.SCAN_TYPE_BLOCKPAGE:
     return satellite.BLOCKPAGE_BIGQUERY_SCHEMA
   if scan_type == satellite.SCAN_TYPE_SATELLITE:
-    full_schema: Dict[str, Any] = {}
-    full_schema.update(SCAN_BIGQUERY_SCHEMA)
-    full_schema.update(satellite.SATELLITE_BIGQUERY_SCHEMA)
-    return full_schema
-  return SCAN_BIGQUERY_SCHEMA
+    return add_schemas(BASE_BIGQUERY_SCHEMA,
```

Comment:
Instead of dynamically creating the schema, define them as constants. Example

```py
SATELLITE_BIGQUERY_SCHEMA = BASE_BIGQUERY_SCHEMA.copy().update(...)
```

Mutations add complexity and currently the HYPERQUACK/SATELLITE_SCHEMA are not a full schema, so the variable name is essentially lying, which hurts readability.

Reply:
Renamed the partial schemas as `ADDITIONAL_HYPERQUACK_BIGQUERY_SCHEMA`/`ADDITIONAL_SATELLITE_BIGQUERY_SCHEMA` and defined the full schemas as constants in `beam_tables.py`. I kept the `_add_schemas` function instead of doing an inline `.copy().update(...)` though to improve readability.